src/components/modals/StaticModalComponent.js

```diff
@@ -16,13 +16,17 @@ type Props = {
   cancel(): void
 }
 class StaticModalComponent extends Component<Props> {
+  reset: number
```

Comment:
if this property is set in `componentDidMount`, then shouldn't this be `reset?: number`